DDTest/CMakeLists.txt

```diff
@@ -58,13 +58,46 @@ if (DD4HEP_USE_GEANT4)
   SET_TESTS_PROPERTIES( t_test_python_import_ddg4 PROPERTIES FAIL_REGULAR_EXPRESSION  "Exception;EXCEPTION;ERROR;Error" )
 
   if(DD4HEP_USE_HEPMC3)
-    ADD_TEST( t_test_ddsim_ion_hepmc3 "${CMAKE_INSTALL_PREFIX}/bin/run_test.sh"
-      ddsim --compactFile=${CMAKE_INSTALL_PREFIX}/DDDetectors/compact/SiD.xml --runType=batch -N=1
-      --outputFile=testSidIons.root --inputFiles ${CMAKE_CURRENT_SOURCE_DIR}/inputFiles/Pythia_output.hepmc
-      --part.userParticleHandler=)
-    # Tests causes G4Exception about: Isomer level 9 may be ambiguous, just a warning
-    # space before Exception intentional!!!
-    SET_TESTS_PROPERTIES( t_test_ddsim_ion_hepmc3 PROPERTIES FAIL_REGULAR_EXPRESSION  "ERROR;Error; Exception" )
+    set(TEST_HEPMC3_EXTENSIONS)
+
+    if(${DD4HEP_HEPMC3_COMPRESSION_SUPPORT})
+      file(ARCHIVE_CREATE
+        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/Pythia_output.hepmc.gz
+        PATHS ${CMAKE_CURRENT_SOURCE_DIR}/inputFiles/Pythia_output.hepmc
```

Comment:
Do you have another hepmc file we can use for theses tests? The tests take 280 seconds each, which is a bit excessive.

Reply:
We can truncate the size of the file. 3 events should be no worse than 9.

Reply:
This is only simulating one event `-N=1`, truncating the file size is not going to change that.

Reply:
Good point. I'm providing a simpler file with geantinos (PDG=0).

Reply:
Are you sure this simulation was slow? I haven't measured locally, but on CI it was and still is about the same as on master.

Reply:
I get the following timings:
```
      Start 14: t_test_ddsim_ion_hepmc3
14/20 Test #14: t_test_ddsim_ion_hepmc3 ..........   Passed   93.48 sec
      Start 15: t_test_ddsim_ion_hepmc3_gz
15/20 Test #15: t_test_ddsim_ion_hepmc3_gz .......   Passed   10.96 sec
      Start 16: t_test_ddsim_ion_hepmc3_xz
16/20 Test #16: t_test_ddsim_ion_hepmc3_xz .......   Passed    8.67 sec
      Start 17: t_test_ddsim_ion_hepmc3_bz2
17/20 Test #17: t_test_ddsim_ion_hepmc3_bz2 ......   Passed    8.67 sec
```

Reply:
Let me rename the tests...

Reply:
```
      Start 14: t_test_ddsim_hepmc3_Pythia_output_hepmc
14/20 Test #14: t_test_ddsim_hepmc3_Pythia_output_hepmc ...   Passed   68.39 sec
      Start 15: t_test_ddsim_hepmc3_geantinos_hepmc_gz
15/20 Test #15: t_test_ddsim_hepmc3_geantinos_hepmc_gz ....   Passed   12.47 sec
      Start 16: t_test_ddsim_hepmc3_geantinos_hepmc_xz
16/20 Test #16: t_test_ddsim_hepmc3_geantinos_hepmc_xz ....   Passed   10.43 sec
      Start 17: t_test_ddsim_hepmc3_geantinos_hepmc_bz2
17/20 Test #17: t_test_ddsim_hepmc3_geantinos_hepmc_bz2 ...   Passed    9.40 sec
```